src/scss/generic/_heading.scss

```diff
@@ -1,22 +1,27 @@
-
+/**
+  * @parent gaiden_css.generic
+  * @stylesheet gaiden_css.generic.headings.css Headings
+  * @demo demo/gaiden-css/generic/headings/demo.html
+**/
 h1,
 h2,
 h3,
 h4,
 h5,
+h6,
 .page-title,
 .section-title,
 .container-title,
 .card-title,
 .info-title {
-  color: get-color(blackNeutral);
+  color: get-color(rastaGray);
   font: {
-    weight: 700;
+    weight: 900;
```

Comment:
I think we need to have more than one case to export values to variables or maps. If there is more than one case, let's move to a map and use some function like `get-weight(bold)`.